player/tests/data/quad.vert

```diff
@@ -0,0 +1,18 @@
+#version 450
+
+out gl_PerVertex {
+    vec4 gl_Position;
+};
+
+const vec2 positions[6] = vec2[6](
```

Comment:
There is an unfortunate bug on SPIRV-Cross version that we are currently using, related to constant arrays like this. So it would be better to just generate the positions in the code.